css/main.scss

```diff
@@ -39,3 +39,19 @@ h2 {
     @include grid-column( 1/3 );
   }
 }
+
+.navigation {
+  padding: 0;
+  margin: 0 0 20px 0;
+
+  li {
+    list-style: none;
+    padding: 0;
+    display: inline-block;
+    margin-right: 20px;
+  }
+
+  a.active {
+    color: #333;
```

Comment:
If you added a new colour I would add it in the pallete file, see here: https://github.com/alphagov/govuk-developers/tree/gh-pages/css/toolkit/colours